www/js/components/LeafletView.jsx

```diff
@@ -65,9 +65,12 @@ const LeafletView = ({ geojson, opts, ...otherProps }) => {
   );
 };
 
-const startIcon = L.divIcon({className: 'leaflet-div-icon-start', iconSize: [18, 18], html: '<div class="leaflet-div-ionicon leaflet-div-ionicon-start">★</div>'});
-const stopIcon = L.divIcon({className: 'leaflet-div-icon-stop', iconSize: [18, 18], html: '<div class="leaflet-div-ionicon leaflet-div-ionicon-stop">⚑</div>'});
-const pointToLayer = (feature, latlng) => {
+const startIcon = L.divIcon({className: 'leaflet-div-icon-start', iconSize: [18, 18], 
+  html: '<svg class="div-icon-start" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2C8.1 2 5 5.1 5 9C5 14.2 12 22 12 22S19 14.2 19 9C19 5.1 15.9 2 12 2M14.5 13L12 11.5L9.5 13L10.2 10.2L8 8.3L10.9 8.1L12 5.4L13.1 8L16 8.3L13.8 10.2L14.5 13Z" /></svg>'});
```

Comment:
Where did these SVG paths come from?

Reply:
I downloaded the svgs from [here](https://pictogrammers.com/library/mdi/) and when I inserted the files this what what was inside. I was thinking directly putting them in was simpler than routing to a file path? But I could put the files in so that it's clearer where things came from

Reply:
Ok. Ideally we'd get them from the "MaterialCommunityIcons" font-family (registered in `angular-react-helper.jsx`). That's the underlying font that React Native Paper is using when it renders icons

But I think if you just add comments in the code explaining where the hardcoded SVGs are from, this is ok for now

Reply:
They are rendered with the "MaterialCommunityIcons" font now, but I had trouble fully centering them in the circles. They are looking fine left-right but not up-down